packages/components/src/components/Pagination/usePagination.tsx

```diff
@@ -0,0 +1,64 @@
+import { useMemo } from "react";
+
+const DOTS = "...";
+
+const range = (start: number, end: number): string[] => {
```

Comment:
Could you also add a JSdoc comment or something? It took me a bit to figure out what this function does.